packages/liveblocks-react/src/liveblocks.tsx

```diff
@@ -631,6 +631,6 @@ export function useLiveblocksContextBundle() {
 export function createLiveblocksContext<
   U extends BaseUserMeta = DU,
   M extends BaseMetadata = never, // TODO Change this to DM for 2.0
->(client: Client<U>): LiveblocksContextBundle<U, M> {
+>(client: OpaqueClient): LiveblocksContextBundle<U, M> {
```

Comment:
These are the two regressions, which degrade the DX when working with the factories.